siddhi-extension-archetype/siddhi-map/src/main/resources/archetype-resources/component/src/test/resources/testng.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
license header? Please correct in all the places

Reply:
According to the code review comment, all files under the archetype-resources folder don't need to put the license header. Because these files are a resource for generating the project using the archetype. Hence I didn't put the license header.